awx/ui_next/public/index.html

```diff
@@ -1,6 +1,15 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <% if (process.env.NODE_ENV === 'production') { %>
+      <script nonce="{{ csp_nonce }}" type="text/javascript">
+        window.NONCE_ID = '{{ csp_nonce }}';
+      </script>
+      <meta
+        http-equiv="Content-Security-Policy"
+        content="default-src 'self'; connect-src 'self' ws: wss:; style-src 'self' 'nonce-{{ csp_nonce }}' *.pendo.io; script-src 'self' 'nonce-{{ csp_nonce }}' *.pendo.io; img-src 'self' *.pendo.io data:;"
+      />
+    <% } %>
```

Comment:
Is it okay to use the same nonce for styles and scripts? Since a nonce can "only be used once," I'm vaguely worried some browser may reject the second one

Reply:
That's a good question. I _think_ we're ok there. 

My understanding is that the nonce should be used only once _per page load_. The way `__webpack_nonce__` works seems to support this, in that it takes one `nonce` value and applies it globally to both `style-src` and `inline-src`.

Reply:
Yep, the nonce is intended as a per-page load nonce, I think (what @jakemcdermott said).

Reply:
I used firefox with those changes and there is no errors reported.